cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,99 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
+  let changedUser;
 
+  beforeEach(() => {
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+
+    cy.task('generateChanges').then((generateChanges) => {
+      changedUser = generateChanges;
+    });
+  });
+
+  it.skip('web-tables should have pagination and selection', () => {
+    cy.get('.-pageInfo')
+      .should('contain', 'Page');
+
+    cy.get('.-next')
+      .should('contain', 'Next');
+
+    cy.get('.-previous')
+      .should('contain', 'Previous');
+
+    cy.get('.-totalPages')
+      .should('be.visible')
+      .should('exist');
+
+    cy.get('select')
+      .should('be.visible')
+      .should('contain', '5 rows')
+      .should('contain', '10 rows')
+      .should('contain', '20 rows')
+      .should('contain', '25 rows')
+      .should('contain', '50 rows')
+      .should('contain', '100 rows')
+      .select('10 rows');
+
+    cy.get('.-previous')
+      .should('not.have.class', 'disabled');
+
+    cy.get('.-previous')
+      .should('not.have.class', 'disabled');
+
+    cy.get('.rt-tr')
+      .should('have.length', '11');
+  });
+
+  it.skip('it should provide user to add workers in table', () => {
+    cy.addWorker(user, 1);
+
+    cy.checkWorker(user);
+  });
+
+  it('it should provide user to delete worker in table', () => {
```

Comment:
this test fails 
![image](https://github.com/mate-academy/qa_cypress_e2e_web_tables/assets/85580286/17ace22d-725e-4fae-bfa5-3fd43c977083)